package.json

```diff
@@ -23,7 +23,9 @@
     "pm2": "^4.2.3"
   },
   "dependencies": {
+    "mailgun-js": "^0.22.0",
     "pg": "^8.0.2",
+    "promisify-call": "^2.0.4",
```

Comment:
is this module necessary? I don't see you using it anywhere. if not, please remove it.